lightspeed/assemblies/assembly_logging-in-out-portal.adoc

```diff
@@ -9,22 +9,24 @@ ifdef::context[:parent-context: {context}]
 :context: logging-in-out-lightspeed-portal
 
 [role="_abstract"]
-Ansible users can use {LightspeedShortName} after the organization administrator has assigned them seat licenses. You access Ansible Lightspeed through the Ansible Lightspeed portal. {LightspeedShortName} uses {RHSSO} ({RHSSOshort}) for authentication, and verifies that you have an assigned seat license. After you log in to the Ansible Lightspeed portal, your assigned user role is displayed on the login screen. 
+
+By default, all Ansible users within an organization that has a trial or paid subscription to both {PlatformNameShort} and {ibmwatsonxcodeassistant} can access {LightspeedShortName}.
+
+You can access Ansible Lightspeed through the link:https://c.ai.ansible.redhat.com/[Ansible Lightspeed portal]. After you enter your {RHSSO} ({RHSSOshort}) account credentials, your account is authenticated and access is granted. Your assigned user role is displayed on the login screen of Ansible Lightspeed portal. 
 
 .User login scenarios
 [cols="50%,50%",options="header"]
 |====
 | *Scenario* | *Result*
-|You are a {RHSSOshort} user with an assigned seat license. Note: This is the typical scenario for accessing as {LightspeedShortName} an Ansible user.| You are routed to the {LightspeedShortName} paid commercial offering.
-|You are a {RHSSOshort} user with an assigned seat license, but your organization administrator has not configured {LightspeedShortName} to connect with {ibmwatsonxcodeassistant}.| You are routed to the {LightspeedShortName} paid commercial offering with a message that your organization administrator has not configured a model for your organization.
-|You are a {RHSSOshort} user without an assigned seat license.|You are identified as a community user and redirected to Red Hat {LightspeedTechPreview}, which is a free community model.
-|Your organization has an Ansible Lightspeed subscription but you do not have an assigned seat license.| You are identified as a community user and redirected to Red Hat {LightspeedTechPreview}, which is a free community model.
+|You are a {RHSSOshort} user. +
+NOTE: This is the typical scenario for accessing as {LightspeedShortName} as an Ansible user.| You are routed to the {LightspeedShortName} paid commercial offering.
```

Comment:
There's an "as" before and after the Lightspeed attribute. The one before the attribute should be removed.